Python/other/SortColors-DutchNationalFlag.py

```diff
@@ -0,0 +1,84 @@
+             
+# PROBLEM STATEMENT- https://leetcode.com/problems/sort-colors/
+
+# Given an array nums with n objects colored red, white, or blue, sort them in-place so that objects of the same color are adjacent, with the colors in the order red, white, and blue.
+# Here, we will use the integers 0, 1, and 2 to represent the color red, white, and blue respectively.
+# In simpler words sort an array containing 0,1,2.
+
+# Solve the problem with a one-pass algorithm using only O(1) constant space?
+
+# The bruteforce approach - Sort the vector in nlogn time
+
+#                                 ************************DUTCH_NATIONAL_FLAG ALGORITHM***********************************
+
```

Comment:
Remove all these extra blank lines.

Reply:
Yes I have removed them. Please review and approve.